pyrokinetics/equilibrium/transp.py

```diff
@@ -191,13 +203,21 @@ def read(
             RZ_coords = np.stack([x.ravel() for x in np.meshgrid(R, Z)], -1)
             psi_RZ = psi_interp(RZ_coords).reshape((nZ, nR)).T
 
+            # Get current and b_axis to determine COCOS
+            # Using 'measured plasma current', PCUR. Should this instead be one of the
+            # following?
+            # - EQ PLASMA CURRENT, PCUREQ (seems to be zero in some files?)
+            # - CALCULATED PLASMA CURRENT, PCURC
+            # - TOTAL PLASMA CURRENT, CUR (units of amps per centimeter^2)
+            I_p = data["PCUR"][time_index] * units.ampere
```

Comment:
In TRANSP there is "PCUR" and "PCURC". In an interpretive run these are the same, but in a predictive run when the current is calculated we would use "PCURC". Probably should switch to that to be safe

```suggestion
            I_p = data["PCURC"][time_index] * units.ampere
```

Reply:
Good to know, I've made the change and edited the preceding comment.